cmd/restic/cmd_copy.go

```diff
@@ -184,57 +185,64 @@ type treeCloner struct {
 	buf          []byte
 }
 
-func (t *treeCloner) copyTree(ctx context.Context, treeID restic.ID) error {
+func (t *treeCloner) copyTree(ctx context.Context, rootTreeID restic.ID) error {
 	// We have already processed this tree
-	if t.visitedTrees.Has(treeID) {
+	if t.visitedTrees.Has(rootTreeID) {
```

Comment:
isn't this check already done by the `skip` function?

Reply:
I've removed the check. Setting up `StreamTrees` to do nothing should require basically no time at all.